js/script.js

```diff
@@ -400,13 +400,35 @@ $(document).ready(function(){
 		}
 	})
 	*/
-	
-	// VERSION FIX ON MOBILE
-	$('.headlinebar .version').click(function(){
-		$('.versionsidebar').toggleClass('active');
-	})
-	
-	
+
+	var outdatedLinks = [
+		'https://duckdb.org/docs/archive/0.6.1/index',
+		'https://duckdb.org/docs/archive/0.5.1/index',
+		'https://duckdb.org/docs/archive/0.4.0/index',
+		'https://duckdb.org/docs/archive/0.3.4/index',
+		'https://duckdb.org/docs/archive/0.3.3/index',
+		'https://duckdb.org/docs/archive/0.3.2/index',
+		'https://duckdb.org/docs/archive/0.3.1/index',
+		'https://duckdb.org/docs/archive/0.3.0/index',
+		'https://duckdb.org/docs/archive/0.2.9/index',
+		'https://duckdb.org/docs/archive/0.2.8/index'
+	  ];
+	  
+	  if (outdatedLinks.includes(window.location.href)) {
```

Comment:
I'd strongly prefer we implement this using Jekyll Liquid templates instead, then we don't have to hardcode any URLs or versions